<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInformationDialog</class>
 <widget class="QDialog" name="GameInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/mgo.png</normaloff>:/res/mgo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>551</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>551</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="playerTab">
    <attribute name="title">
     <string>Player/Rule</string>
    </attribute>
    <widget class="QGroupBox" name="whiteGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>White</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rank</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="whitePlayer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="whiteRank">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Team</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="whiteTeam">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="blackGroupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Black</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rank</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="blackPlayer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="blackRank">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Team</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="blackTeam">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ruleGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>531</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Rule</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Komi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="komi">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="result">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Handicap</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="handicap">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>OverTime</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="time">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="overTime">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>531</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>When and Where</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Game Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Place</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gameName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="place">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>52</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Round</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="round">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>52</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Event</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="event">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="optionTab">
    <attribute name="title">
     <string>Other</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>531</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Other</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Source</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Game Comment</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Copyright</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Annotation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="source">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="gameComment">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="copyright">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="annotation">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>421</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>52</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>opening</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="opening">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>whitePlayer</tabstop>
  <tabstop>whiteRank</tabstop>
  <tabstop>whiteTeam</tabstop>
  <tabstop>blackPlayer</tabstop>
  <tabstop>blackRank</tabstop>
  <tabstop>blackTeam</tabstop>
  <tabstop>komi</tabstop>
  <tabstop>handicap</tabstop>
  <tabstop>result</tabstop>
  <tabstop>time</tabstop>
  <tabstop>overTime</tabstop>
  <tabstop>gameName</tabstop>
  <tabstop>date</tabstop>
  <tabstop>round</tabstop>
  <tabstop>place</tabstop>
  <tabstop>event</tabstop>
  <tabstop>user</tabstop>
  <tabstop>source</tabstop>
  <tabstop>opening</tabstop>
  <tabstop>gameComment</tabstop>
  <tabstop>copyright</tabstop>
  <tabstop>annotation</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameInformationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameInformationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
